Development/nmos/test/video_jxsv_test.cpp

```diff
@@ -56,3 +57,128 @@ a=fmtp:112 packetmode=0; profile=High444.12; level=1k-1; sublevel=Sublev3bpp; de
         BST_CHECK_EQUAL(expected_line, actual_line);
     } while (!expected.fail() && !actual.fail());
 }
+
+////////////////////////////////////////////////////////////////////////////////////////////
+BST_TEST_CASE(testSdpParametersVideoJpegXs)
+{
+    std::pair<nmos::sdp_parameters, nmos::video_jxsv_parameters> example{
+        {
+            U("example"),
```

Comment:
I see this example corresponds exactly with the example `a=fmtp:` line in the test case above, so that's nice. 👍